tests/wpt/css-tests/selectors-3_dev/xhtml1/css3-selectors-lang-014.xht

```diff
@@ -27,7 +27,7 @@ This tests a detail related to :lang support. If :lang is not supported, a messa
 <script>
 test(function() {
 assert_equals(document.getElementById('colonlangcontroltest').offsetWidth, 0)
-assert_equals(document.getElementById('box').offsetWidth, 50);
+assert_equals(document.getElementById('box').offsetWidth, 100);
```

Comment:
You can't just simply change the tests like that to fit your behavior. These are tests that are shared among all browsers. What is the actual problem here that makes you want to change the assertion?